django/contrib/admin/static/admin/js/actions.js

```diff
@@ -179,23 +179,16 @@
                 }
             });
         }
-    };
 
-    // Call function fn when the DOM is loaded and ready. If it is already
-    // loaded, call the function now.
-    // http://youmightnotneedjquery.com/#ready
-    function ready(fn) {
-        if (document.readyState !== 'loading') {
-            fn();
-        } else {
-            document.addEventListener('DOMContentLoaded', fn);
-        }
-    }
+        // Enforce clearing checkboxes.
+        checker(actionCheckboxes, options, false);
 
-    ready(function() {
+    };
+
+    window.onpageshow = function() {
```

Comment:
I replaced the `ready` or the `load` functions by `onpageshow` because Safari doesn't trigger any of them after do a browser back